### ISSUE TITLE
[Bug] 23w45a- Solar Panel Issues

### ISSUE THREAD

**Issue by Crimson-Artist** [tsunamayo/Starship-EVO#5351]
Solar Panels work now but they still have some bugs.

1. **Panel efficiency applies to the wrong side.** For some reason the solar panels energy efficiency is greater when the thin edge of the panel is facing the sun as oppose to the flat side.  
![20231114083122_1](https://github.com/tsunamayo/Starship-EVO/assets/112352428/94b42ced-1528-4160-9008-f37b826d3397)
![20231114083055_1](https://github.com/tsunamayo/Starship-EVO/assets/112352428/a6ac65f4-9dd7-4d07-9e56-76cbe60c2260)
2. **Energy bar does not display when solar panels are powering an entity.** When the primary power source of an entity is solar panels the energy bar that is displayed on the lower left side of the HUD does not display any activity. The solar panels are clearly delivering energy as it is possible to gain a negative energy balance warning when a solar panel array is too small to supply a ship. 
![20231114083143_1](https://github.com/tsunamayo/Starship-EVO/assets/112352428/aa6c66b0-59ea-4772-8b1c-fb264d9afae3)
![20231114083148_1](https://github.com/tsunamayo/Starship-EVO/assets/112352428/0766cabe-8861-47fe-add4-9cc1f1009cd3)

**Comment by tsunamayo** [comment]
@Crimson-Artist Hi, thanks good catch!

**Comment by Crimson-Artist** [comment]
@tsunamayo Thank you. 

Also, after i posted this I did some additional testing and found that Solar Panels still generate power even when they are completely covered up by other blocks. because of this I was able to stack several solar panels on top of each other using the Copy-paste tool. That solar panel stack collectively was able to generate far more energy then a similarly sized reactor. Seems kinda broken if you ask me
